pkg/util/utils_linux.go

```diff
@@ -52,3 +53,18 @@ func FindDeviceNodes() (map[string]string, error) {
 
 	return nodes, nil
 }
+
+// CheckRootlessUIDRange checks the uid within the rootless container is in the range from /etc/subuid
+func CheckRootlessUIDRange(uid int) error {
+	uids, _, err := rootless.GetConfiguredMappings()
```

Comment:
I think there should be a remote check here and return nil if remote.

Reply:
Disabled this test under cmd/podman if it's remote